fe1-web/src/features/popcha/screens/PopchaScanner.tsx

```diff
@@ -33,7 +37,7 @@ const PopchaScanner = () => {
   const generateToken = PopchaHooks.useGenerateToken();
 
   const [showScanner, setShowScanner] = useState(false);
-  const [textScanned, setTextScanned] = useState('');
+  const [showInputModal, setInputModalIsVisible] = useState(false);
```

Comment:
```suggestion
  const [isInputModalVisible, setIsInputModalVisible] = useState(false);
```
I think it is better if the names are the same with a set in front of the setter.